main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -225,4 +236,59 @@ static class EqualShapeDistanceDiffCoordinatesNotice extends ValidationNotice {
           getDistanceMeters(current.shapePtLatLon(), previous.shapePtLatLon());
     }
   }
+
+  /**
+   * Two consecutive points have equal `shape_dist_traveled` and different lat/lon coordinates in
+   * `shapes.txt` and the distance between the two points is less than 1.11m.
+   *
+   * <p>When sorted by `shape.shape_pt_sequence`, the values for `shape_dist_traveled` must increase
```

Comment:
Why is an error mentioned when it's a warning? Shouldn't the 1.11 m be mentioned somewhere?